tests/PKSim.Tests/IntegrationTests/ImportPopulationFactorySpecs.cs

```diff
@@ -38,4 +47,21 @@ public void should_have_imported_3_individuals()
          _population.NumberOfItems.ShouldBeEqualTo(3);
       }
    }
+
+   public class When_importing_a_population_file_containing_two_column_with_the_same_entity_path : concern_for_ImportPopulationFactory
+   {
+      public override void GlobalContext()
+      {
+         base.GlobalContext();
+         _population = sut.CreateFor(new[] { DomainHelperForSpecs.PopulationFilePathFor("pop_3_duplicated_column") }, _individual, new CancellationToken()).Result;
+      }
+
+      [Observation]
+      public void should_show_an_error_to_the_user_explaning_that_a_column_duplicate_and_specify_which_one_is_duplicated()
+      {
+         var populationfIle = _population.Settings.AllFiles.ElementAt(0);
```

Comment:
Funny. With the changes from yesterday in CSV Export (and update to newest component), this is workign out of the box

Here is the message that I am geting
![image](https://user-images.githubusercontent.com/1041237/38325864-3dcd9bb6-3812-11e8-8afd-a9e97f412714.png)